src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -2095,63 +2535,186 @@ await RunFormattingTestAsync(
+                    <button @onclick="@(e=>currentCount=4)">Update to 4</button>
+                    <button @onclick="e=>currentCount=5">Update to 5</button>
+
+                    @code {
+                        private int currentCount=0;
+
+                        private void IncrementCount()
+                        {
+                            currentCount++;
+                        }
+                    }
+                    """,
+            expected: """
+                    <p>Current count: @currentCount</p>
+
+                    <button @onclick="IncrementCount">Increment</button>
+                    <button @onclick="@(e => currentCount = 4)">Update to 4</button>
+                    <button @onclick="e => currentCount = 5">Update to 5</button>
+
+                    @code {
+                        private int currentCount = 0;
+
+                        private void IncrementCount()
+                        {
+                            currentCount++;
+                        }
+                    }
+                    """,
+            fileKind: FileKinds.Component);
+    }
+
+    [FormattingTestFact]
+    public async Task FormatEventCallbackAttributes()
+    {
+        await RunFormattingTestAsync(
+            input: """
+                    @using Microsoft.AspNetCore.Components.Forms;
+
+                    <p>Current count: @currentCount</p>
+
+                    <InputText ValueChanged="IncrementCount">Increment</InputText>
+                    <InputText ValueChanged="@(e=>currentCount=4)">Update to 4</InputText>
+                    <InputText ValueChanged="e=>currentCount=5">Update to 5</InputText>
+
+                    @code {
+                        private int currentCount=0;
+
+                        private void IncrementCount()
+                        {
+                            currentCount++;
+                        }
+                    }
+                    """,
+            expected: """
+                    @using Microsoft.AspNetCore.Components.Forms;
+
+                    <p>Current count: @currentCount</p>
+
+                    <InputText ValueChanged="IncrementCount">Increment</InputText>
+                    <InputText ValueChanged="@(e => currentCount = 4)">Update to 4</InputText>
+                    <InputText ValueChanged="e => currentCount = 5">Update to 5</InputText>
+
+                    @code {
+                        private int currentCount = 0;
+
+                        private void IncrementCount()
+                        {
+                            currentCount++;
                         }
-                    </div>
+                    }
                     """,
             fileKind: FileKinds.Component);
     }
 
     [FormattingTestFact]
-    [WorkItem("https://github.com/dotnet/razor/issues/8606")]
-    public async Task FormatAttributesWithTransition()
+    public async Task FormatBindAttributes()
```

Comment:
The diff on this is wonky, but `FormatBindAttributes` is an entirely new test and `FormatAttributesWithTransition` still exists on line 2551 above